.github/workflows/integration.yml

```diff
@@ -0,0 +1,243 @@
+          echo "Retrying in 5s..."
+          sleep 5
+        done
+        TOKEN=$(cat ~/.fleet/config| grep token | awk '{ print $2 }')
+        echo "::set-output name=token::$TOKEN"
+      
+  orbit-macos:
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        orbit-channel: [ 'stable', 'edge' ]
+        osqueryd-channel: ['stable', 'edge' ]
+    runs-on: macos-latest
+    needs: [gen, login]
+    steps:
+    - name: Install dependencies
+      run: |
+        npm install -g fleetctl
+        fleetctl config set --address ${{ needs.gen.outputs.address }} --token ${{ needs.login.outputs.token }}
+
+    - name: Install Orbit
+      run: |
+        sudo hostname macos-orbit-${{ matrix.orbit-channel }}-osqueryd-${{ matrix.osqueryd-channel }}
+        SECRET_JSON=$(fleetctl get enroll_secret --json --debug)
+        echo $SECRET_JSON
+        SECRET=$(echo $SECRET_JSON | jq -r '.spec.secrets[0].secret')
+        echo "Secret: $SECRET"
+        echo "Hostname: $(hostname -s)"
+        fleetctl package --type pkg --fleet-url=${{ needs.gen.outputs.address }} --enroll-secret=$SECRET --orbit-channel=${{ matrix.orbit-channel }} --osqueryd-channel=${{ matrix.osqueryd-channel }}
+        sudo installer -pkg fleet-osquery.pkg -target /
+        until fleetctl get hosts | grep -iF $(hostname -s);
+        do
+          echo "Awaiting enrollment..."
+          sleep 1
+        done
+
+  orbit-ubuntu:
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        orbit-channel: [ 'stable', 'edge' ]
+        osqueryd-channel: ['stable', 'edge' ]
+    runs-on: ubuntu-latest
+    needs: [gen, login]
+    steps:
+    - name: Install dependencies
+      run: |
+        npm install -g fleetctl
+        fleetctl config set --address ${{ needs.gen.outputs.address }} --token ${{ needs.login.outputs.token }}
+
+    - name: Download binaries
+      uses: dawidd6/action-download-artifact@09385b76de790122f4da9c82b17bccf858b9557c #v2.16.0
+      with:
+        workflow: build-binaries.yaml
+        branch: main
+        name: build
+        path: build
+        check_artifacts: true
+
+    - name: Install Orbit
+      run: |
+        sudo hostname ubuntu-orbit-${{ matrix.orbit-channel }}-osqueryd-${{ matrix.osqueryd-channel }}
+        chmod +x ./build/fleetctl
+        SECRET_JSON=$(fleetctl get enroll_secret --json --debug)
+        echo $SECRET_JSON
+        SECRET=$(echo $SECRET_JSON | jq -r '.spec.secrets[0].secret')
+        echo "Secret: $SECRET"
+        echo "Hostname: $(hostname -s)"
+        ./build/fleetctl package --type deb --fleet-url=${{ needs.gen.outputs.address }} --enroll-secret=$SECRET  --orbit-channel=${{ matrix.orbit-channel }} --osqueryd-channel=${{ matrix.osqueryd-channel }}
+        sudo dpkg -i fleet-osquery*
+        until fleetctl get hosts | grep -iF $(hostname -s);
+        do
+          echo "Awaiting enrollment..."
+          sudo systemctl status orbit.service || true
+          sleep 1
+        done
+
+  orbit-windows-build:
```

Comment:
Why is there a build job `orbit-windows-build` and then a separate `orbit-windows` install job?